docs/source/zero-code-change-limitations.rst

```diff
@@ -1,21 +1,32 @@
-cuml.accel: Known Limitations
-=============================
+Known Limitations
+-----------------
 
 General Limitations
--------------------
+~~~~~~~~~~~~~~~~~~~
 
 TODO(wphicks): Fill this in
 TODO(wphicks): Pickle
 
 Algorithm-Specific Limitations
-------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 TODO(wphicks): Fills these in. Document when each will fall back to CPU, how to
 assess equivalence with CPU implementations, and significant differences in
 algorithm, as well as any other known issues.
 
+
 ``sklearn.cluster.KMeans``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+The default initialization algorithm used by ``cuml.accel`` is similar, but different.
```

Comment:
Can you either give a bit more information or link to somewhere the reader can get more information on these differences? Perfectly fine if that's just a link to source code if we don't have something better.

Reply:
I expanded it a bit and now referring people to the docs of `KMeans`.

@cjnolet is https://arxiv.org/abs/1203.6402 the paper to refer people to if they want to get more details on how the initialisation of cluster centers is done in cuml? Is there a reason to not link it from the docs? Basically, wondering why it isn't already linked and if there is a reason for that